src/random.cpp

```diff
@@ -17,7 +17,7 @@
 #include <sync.h>     // for WAIT_LOCK
 #include <utiltime.h> // for GetTime()
 
-#include <stdlib.h>
+#include <unistd.h>
```

Comment:
Why is this change necessary?
There is a conditional include in `L37-L39`.

Reply:
Right, the new include is not necessary.